src/stylesheets/components/_hero.scss

```diff
@@ -2,36 +2,45 @@
 // ==========================
 
 .usa-hero {
-  @include u-padding-y($site-margins);
+  @include typeset;
+  @include u-padding-y(units($theme-site-margins));
   background-image: url('#{$theme-image-path}/hero.png');
   background-position: center;
   background-size: cover;
 
   .usa-button {
-    color: $color-white;
+    color: color('white');
+  }
+
+  p {
+    @include typeset-p;
   }
 }
 
 .usa-section-dark {
   .usa-hero-link {
-    color: $color-primary-alt;
+    color: color('accent-cool');
```

Comment:
@maya what's this used for? I don't see it in the example

Reply:
In case someone wants to add a link. It's not in the example, but may have been at one point.

<img width="334" alt="screen shot 2018-09-12 at 11 38 07 am" src="https://user-images.githubusercontent.com/5249443/45445917-78968680-b680-11e8-9a8e-a8eae30ef3d9.png">

Reply:
I think the default link color for this section is fine though, so removed in 5c1a955.
<img width="302" alt="screen shot 2018-09-12 at 11 42 54 am" src="https://user-images.githubusercontent.com/5249443/45446155-02465400-b681-11e8-9b2e-7cb167762c68.png">